### PR TITLE
Wysiwyg

### PR DESCRIPTION
**Pull request by emanuele45**
That's all I think (well, optimization and bug fixing will follow, but the main thing is there and working).

A quick look at the last commits (where I had to add another jQuery plugin and I reverted html_to_bbc and bbc_to_html for ompatibility) wouldn't hurt.

### CODE REVIEW

File: Sources/Subs-Editor.php [hunk 1]
```diff
@@ -1712,21 +1269,122 @@ function spellCheckDone()
...
 			);
 			$context['bbc_tags'][count($context['bbc_tags']) - 1][] = array(
-				'image' => 'toggle',
 				'code' => 'toggle',
-				'before' => '',
 				'description' => $txt['toggle_view'],
 			);
 		}
 
+		// Generate a list of buttons that shouldn't be shown - this should be the fastest way to do this.
```
Comment (joshuaadickerson) [inline Sources/Subs-Editor.php:1675]: It is late but I can't sleep so forgive me if I am off on this.

Using the disabled tag means that it won't parse them, not just not show them. Is that what you wanted to do here? If so, there should be a hook somewhere to add to this list.

File: Themes/default/languages/index.english.php [hunk 2]
```diff
@@ -8,7 +8,7 @@
 //   - http://www.php.net/function.pspell-new
 //   - http://www.php.net/function.setlocale
 // Again, SPELLING SHOULD BE '' 99% OF THE TIME!!  Please read this!
-$txt['lang_locale'] = 'en_US';
+$txt['lang_locale'] = 'it_US';
```
Comment (joshuaadickerson) [inline Themes/default/languages/index.english.php]: You changed the English locale to Italian?

File: Sources/Subs-Editor.php (hunk 1 above)
Comment (emanuele45) [inline Sources/Subs-Editor.php:1675]: No problem! ;D

Why a hook?
This is the $modSettings['disabledBBC'] code that has been moved up a bit, nothing more (well, plus a bit of "compatibility" thing to match the codes from the editor and those from SMF).
You set the disabled bbc from the admin panel.

...fact is that this code would not disable predefined tags...but even the aviour of the current editor (i.e. strip the tag) is not exactly the best...

meh this needs to be fixed somehow.

File: Themes/default/languages/index.english.php (hunk 2 above)
Comment (emanuele45) [inline Themes/default/languages/index.english.php]: lol

/me has left around some debugging code...nothing new! :P

**Pull request merged**